docs/data/external-oasis-repos.md

```diff
@@ -1,50 +1,52 @@
-Procedures for handling OASIS externally-hosted CVMFS repositories
-==================================================================
-
-**Technical information contained here is believed to be reliable but administrative procedures, particularly as concerns organizational entities outside the OSG, should be considered unapproved proposals.**
-
+Setting up a Software Repository
+================================
 
 About OASIS
 -----------
 
-OASIS is the OSG Application Software Installation Service. It is the recommended method to install software on the Open Science Grid. It is implemented using CernVM FileSystem (CVMFS) technology.
+OASIS (the **OSG** **A**pplication **S**oftware **I**nstallation **S**ervice) is an infrastructure, based on
+[CVMFS](https://cvmfs.readthedocs.io), for distributing software throughout the OSG.  Once software is installed into
+an OASIS repository, it should be available across about 90% of the OSG within an hour or two.
 
-About this Document
--------------------
+OASIS consists of keysigning infrastructure, a content distribution network (CDN), and a shared CVMFS repository that
+is hosted by the OSG.  Many use cases will be covered by utilizing the [shared repository](update-oasis); this documen
+covers how to setup and host your own CVMFS _repository server_.  This server will  distribute softwarevia OASIS, but
```

Comment:
* I prefer "how to install, configure, and run" over "setup and host"
* two spaces between "will" and "distribute"
* softwarevia typo